crates/agent/src/publications/specs.rs

```diff
@@ -616,8 +616,8 @@ fn extract_spec_metadata<'a>(
             image_parts = Some(split_tag(&config.image));
 
             for binding in &capture.bindings {
-                if let Some(target) = binding.target.as_ref() {
-                    writes_to.push(target.as_ref());
+                if !binding.disable {
```

Comment:
do this for `reads_from` below, too? Both derivations and materializations need it.

Reply:
:+1: good catch!